src/js/components/ContainerSettingsComponent.jsx

```diff
@@ -238,6 +238,16 @@ var ContainerSettingsComponent = React.createClass({
       return null;
     }
 
+    if (this.props.fields.dockerNetwork === ContainerConstants.NETWORK.HOST) {
+      return (
+        <div className="help-block">
+          While host networking is selected you cannot define port mappings.
+          <br />
+          Consider to modify the applications port configuration.
```

Comment:
Most of our error messages don't address the user as "you" (there is one case, in the recently-merged scaling error)

I think we could simply offer
`Ports may only be mapped in bridged network mode`

But if we want to point users towards the port field, how about
`Use the ports field to assign ports in host network mode`